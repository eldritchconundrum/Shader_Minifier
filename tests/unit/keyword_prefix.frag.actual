/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef KEYWORD_PREFIX_FRAG_ACTUAL_
# define KEYWORD_PREFIX_FRAG_ACTUAL_

const char *keyword_prefix_frag =
 "void d()"
 "{}"
 "void i()"
 "{"
   "d();"
 "}";

#endif // KEYWORD_PREFIX_FRAG_ACTUAL_
