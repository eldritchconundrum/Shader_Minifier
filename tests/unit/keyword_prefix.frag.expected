/* File generated with Shader Minifier 1.1.6
 * http://www.ctrl-alt-test.fr
 */
#ifndef KEYWORD_PREFIX_FRAG_EXPECTED_
# define KEYWORD_PREFIX_FRAG_EXPECTED_

const char *keyword_prefix_frag =
 "void d()"
 "{}"
 "void i()"
 "{"
   "d();"
 "}";

#endif // KEYWORD_PREFIX_FRAG_EXPECTED_
